src/main/java/fi/otavanopisto/kuntaapi/server/index/AbstractIndexHander.java

```diff
@@ -91,6 +91,8 @@ private TransportClient createTransportClient(String[] hosts, String clusterName
     try {
       TransportClient transportClient;
       
+      System.out.println(String.format("Cluster %s, %s", clusterName, StringUtils.join(hosts, ",") ));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)